### PR TITLE
HP-894 studies changes

### PR DESCRIPTION
**Pull request by LunaSquee**

### CODE REVIEW

**Comment by LunaSquee** [comment]
!! DO NOT MERGE INTO DEVELOP !!
Because of the develop conflict, this PR can only be used for release. I will manually merge this branch into develop and deal with the consequences.

File: angular-fe/src/app/modules/study/components/study-intro/study-intro.component.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+@import "colors.scss";
+
+.large-title {
+  font-weight: 500;
+  font-size: 24px;
```
Comment (mitka0260) [inline angular-fe/src/app/modules/study/components/study-intro/study-intro.component.scss:5]: We use rem, not px

File: angular-fe/src/app/modules/study/components/study-intro/study-intro.component.scss [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+  line-height: 30px;
```
Comment (mitka0260) [inline angular-fe/src/app/modules/study/components/study-intro/study-intro.component.scss:6]: We use rem, not px

File: angular-fe/src/app/modules/study/containers/study-detail/study-detail.component.html [hunk 3]
```diff
@@ -40,8 +40,14 @@ <h1>{{study.title}}</h1>
...
                     <accordion-item [title]="study?.fieldAccordion?.entity?.fieldStudyPageAccordionTitle">
                         <ng-template>
-                            <p class="bold" [innerHTML]="study?.fieldAccordion?.entity?.fieldStudyAccordionIntro"></p>
-                            <p [innerHTML]="study?.fieldAccordion?.entity?.fieldStudyAccordionContent"></p>
+                            <ng-container *ngIf="!study.fieldAccordion.entity.fieldBody">
+                                <p [innerHTML]="study?.fieldAccordion?.entity?.fieldStudyAccordionIntro"></p>
+                                <p [innerHTML]="study?.fieldAccordion?.entity?.fieldStudyAccordionContent"></p>
+                            </ng-container>
+                            <p
+                                *ngIf="study.fieldAccordion.entity.fieldBody"
```
Comment (mitka0260) [inline angular-fe/src/app/modules/study/containers/study-detail/study-detail.component.html:48]: Maybe *ngIf="study?.fieldAccordion?.entity?.fieldBody"

File: angular-fe/src/app/modules/study/containers/study-detail/study-detail.component.html [hunk 4]
```diff
@@ -40,8 +40,14 @@ <h1>{{study.title}}</h1>
                 <accordion [collapsible]="false" *ngIf="study?.fieldAccordion?.entity?.fieldStudyPageAccordionTitle">
                     <accordion-item [title]="study?.fieldAccordion?.entity?.fieldStudyPageAccordionTitle">
                         <ng-template>
-                            <p class="bold" [innerHTML]="study?.fieldAccordion?.entity?.fieldStudyAccordionIntro"></p>
-                            <p [innerHTML]="study?.fieldAccordion?.entity?.fieldStudyAccordionContent"></p>
+                            <ng-container *ngIf="!study.fieldAccordion.entity.fieldBody">
```
Comment (mitka0260) [inline angular-fe/src/app/modules/study/containers/study-detail/study-detail.component.html:43]: Maybe *ngIf="!study?.fieldAccordion?.entity?.fieldBody"

**Review by mitka0260 — APPROVED** [review]

File: angular-fe/src/app/modules/study/containers/study-detail/study-detail.component.html (hunk 4 above)
Comment (LunaSquee) [reply angular-fe/src/app/modules/study/containers/study-detail/study-detail.component.html:43]: You're already checking if all of those exist in the ngIf of the accordion itself. No need to check any further after that.

**Pull request merged**